### ISSUE TITLE
Hi! I have some advice about the improvement of this project.

### ISSUE THREAD

**Issue by yjh0410** [MarkAny-Vision-AI/CenterNeXt#11]
@developer0hye 
First, I really like this CenterNeXT!

I have a immature advice to improve this project. (Of course, I can't be sure that the suggestion will work.)

I always consider how to make centernet converge faster. Whether it's 140epoch or 200epoch, I think it's too long. We have known that some advanced detectors like RetinaNet，FCOS, YOLOF, etc need to be trained with only 12 epochs(standard 1x schedule). Of course, if we try 1x schedule, we only use RandomHFlip as the augmentation method. 

I think it is possible to make centernet converge by 1x schedule as long as we find a appropriate method. TTF-Net achieves this, but I feel like that work a bit spoils the elegance of the centernet framework. 

As far as I know, some of the necessary conditions for 1x to converge includes ‘freeze stem and first stage of backbone’, only RandomHFlip, and a large learning rate (like 0.01 with batchsize = 16). If we want to use more augmentations like RandomCrop, ColorJitter and Mosaic, we need to train model with more epochs such as 3x schedule (36 epoch). 

In addition, I think we might use multiple positive samples to achieve better performance. In original CenterNet, there is only one positive sample of each groundtruth. Maybe we can consider 3x3 or larger neighborhood of center point. 

How do you think the above suggestions? Do you think the CenterNet can converge faster?

**Comment by developer0hye** [comment]
@yjh0410 
Thank you for your advice!

I agree with you. The training schedule of CenterNet is too long.

First, we want to reproduce the results in the paper on COCO before we speed up the convergence of this reproduced model!

**Comment by yjh0410** [comment]
@developer0hye 
You are welcome！

Looking forward to the faster convergence of your CenterNext ！
